test/mitmproxy/addons/test_proxyauth.py

```diff
@@ -109,6 +121,24 @@ def test_check():
         )
         assert not up.check(f)
 
+        ctx.configure(
```

Comment:
Yeah, indeed I connect to an already set up LDAP server, to be honest running a small ldap locally is possible, it just requires automatic installation and configuration (and it is a bit of pain even with simple solution like openldap) which is why I chose to use an already existing one